### PR TITLE
Clarify testing workflows on remote Galaxy servers.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

**Comment by jmchilton** [comment]
I would normally just merge a small PR like this myself, but I'm linking to an open source project under the GalaxyWorks Github tree so I don't want to be the one to merge this - any chance someone else can review and decide if it is appropriate @mvdbeek @bgruening?

File: docs/test_format.rst [hunk 1]
```diff
@@ -312,7 +312,21 @@ option where integer is the incremental increase in seconds for every request.
...
+using ``galaxy-tool-test --help``. If you're interested in running all the tool tests corresponding to
+a workflow on a running server, check out the galaxy-workflow-tool-tests_ project that is a wrapper around
+galaxy-tool-test that has all the same options but that filters to the tool tests to just run those
+from a specific workflow.
+
+This engine can also be used to test workflows already available in the running Galaxy instance.
+While you don't need to download and synchronize the target workflow on your local filesystem, you
+do need to provide a path to find the test definition and test data paths.
+
+An example of doing this is included in Planemo's test data. The workflow test definition
```
Comment (bgruening) [inline docs/test_format.rst:324]: link to the test-data?

File: docs/test_format.rst [hunk 2]
```diff
@@ -312,7 +312,21 @@ option where integer is the incremental increase in seconds for every request.
...
+``wf11-remote.gxwf-test.yml`` exists but no file ``wf11-remote.gxwf.yml`` exists. The workflow
```
Comment (bgruening) [inline docs/test_format.rst:325]:
```suggestion
``wf11-remote.gxwf-test.yml`` exists but no workflow file ``wf11-remote.gxwf.yml`` exists. The workflow
```

File: docs/test_format.rst [hunk 3]
```diff
@@ -312,7 +312,21 @@ option where integer is the incremental increase in seconds for every request.
...
+is assumed to already exist in some Galaxy server - say with id ``123abc``. Then ``planemo test``
```
Comment (bgruening) [inline docs/test_format.rst:326]:
```suggestion
is assumed to already exist in some Galaxy server - say with id ``99113b2b119318e1``. Then ``planemo test``
```

its a little bit more realistic

**Review by bgruening — COMMENTED** [review]
Some comments inline.

**Comment by jmchilton** [comment]
Rebased with those changes, thanks for the detailed review @bgruening!

**Review by bgruening — APPROVED** [review]
Thanks John!

**Pull request merged**